MAUI/ImageEditor/toolbar-customization.md

```diff
@@ -231,6 +229,105 @@ Change the default colors of the `ColorPalette` in the toolbar.
 
 N> Color palette colors are common for Text, Shape and Pen.
 
+### Change the Toolbar Item to Enable or Disable State
+
+Enable or disable the toolbar items by using the `IsEnabled` property in the `ImageEditorToolbarItem` class.
+
+{% tabs %}
+
+{% highlight C# %}
+
+SfImageEditor imageEditor = new SfImageEditor();
+imageEditor.Source = ImageSource.FromFile("image.png");
+
+ImageEditorToolbar headerToolbar = imageEditor.Toolbars[0];
+ImageEditorToolbarGroupItem browseGroup = (ImageEditorToolbarGroupItem) headerToolbar.ToolbarItems[0];
+ImageEditorToolbarItem browseItem = browseGroup.Items.FirstOrDefault();
+browseItem.IsEnabled = false;
+
+ImageEditorToolbarGroupItem saveGroup = (ImageEditorToolbarGroupItem) headerToolbar.ToolbarItems[2];
+ImageEditorToolbarItem saveItem = saveGroup.Items.FirstOrDefault(i => i.Name == "Save");
+saveItem.IsEnabled = false;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+### Customize the Toolbar Item View
+
+To customize the view of toolbar items, specify their icon names and set the view using the `View` property in the `ImageEditorToolbarItem`.
+
+{% tabs %}
+
+{% highlight C# %}
```

Comment:
Change this code

SfImageEditor imageEditor = new SfImageEditor();
imageEditor.Source = ImageSource.FromFile("image.png");

ImageEditorToolbar headerToolbar = imageEditor.Toolbars[0];
ImageEditorToolbarGroupItem saveGroup = (ImageEditorToolbarGroupItem)headerToolbar.ToolbarItems[2];
ImageEditorToolbarItem saveItem = saveGroup.Items.FirstOrDefault(i => i.Name == "Save");

Image image = new Image();
image.Source = ImageSource.FromFile("save.jpg");
saveItem.View = image;

Reply:
updated.